pom.xml

```diff
@@ -66,7 +66,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>${jackson.version}</version>
+            <version>2.9.10.1</version>
```

Comment:
Skal denne ikke settes i properties, felles for alle jackson versions?

Reply:
Det er kun jackson-databind som har fått en sikkerhetspatch. Resten er fortsatt 2.9.10. Syntest også det var merkelig

Reply:
Aha, daså :)